src/components/bookCatagory/BookCatagory.js

```diff
@@ -0,0 +1,9 @@
+import './BookCatagory.css';
+
+const BookCatagory = () => (
+  <div>
+    <button type="button">Check Status</button>
+  </div>
```

Comment:
- [ ] It would be nice if you put this button into another component and import that component in this **Category** or **BookCategory** component 👍 to make your components reusable, second you would make more components in the future so it would be better if you follow this approach 😊